src/json/pull_parser.cr

```diff
@@ -259,42 +363,62 @@ class JSON::PullParser
     value
   end
 
+  # Reads the next lexer's token.
+  #
+  # Contrary to `read_raw`, it does not read a full value.
+  # For example if the next token is the beginning of an array, it will stop there, while `read_raw` would have read the whole array.
   def read_next
     read_next_internal
     @kind
   end
 
+  # Reads a boolean value and returns it. If the value is not a boolean, returns nil.
```

Comment:
```suggestion
  # Reads a `Bool` value and returns it. If the value is not a boolean, returns nil.
```

Could you add backticks to the types below as well? That creates links to their API docs.